dashboard/src/main/java/no/sikt/nva/monitoring/model/CloudWatchWidget.java

```diff
@@ -1,7 +1,27 @@
 package no.sikt.nva.monitoring.model;
 
+import java.util.Objects;
 import no.unit.nva.commons.json.JsonSerializable;
+import nva.commons.core.JacocoGenerated;
 
 public record CloudWatchWidget<I> (String type, I properties, int height, int width, int x, int y) implements JsonSerializable {
 
+    @JacocoGenerated
+    @Override
```

Comment:
override the equals method, as I don't think it is vital for checking the x, y, width and height properties.

Reply:
If we want to update the layout, would this cause it to not update?

Reply:
Nope, it is only when asserting that the widgets are created during tests.